doc/installation.rst

```diff
@@ -76,148 +90,191 @@ On macOS, open the disk image and drag Spyder to your :guilabel:`Applications` f
+This is our recommended installation method on Linux and for users with third-party Spyder plugins, as support for both of these in our standalone installers is still under active development.
 
-Spyder is included by default in the `Anaconda Python distribution`_, which comes with everything you need to get started in an all-in-one package, and is our recommended installation method on Linux (and supported on the other platforms too).
+.. _Anaconda: https://www.anaconda.com/products/distribution
+.. _Miniconda: https://conda.io/miniconda.html
+.. _Miniforge/Mambaforge: https://github.com/conda-forge/miniforge
 
-.. _Anaconda Python distribution: https://www.anaconda.com/products/distribution
 
+.. _install-anaconda-running:
 
 Running with Anaconda
 ~~~~~~~~~~~~~~~~~~~~~
 
-To run Spyder after installing it with Anaconda, the recommended method on Windows is to launch it via the Start menu shortcut.
-On other platforms, open Anaconda Navigator, scroll to Spyder under ``Home``, and click ``Launch``.
+.. note::
+
+   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is a chance of dependency conflicts or a broken Spyder installation.
+   Therefore, we recommend :ref:`installing Spyder into a new Conda environment <install-conda-installing>` to avoid all these issues.
+
+To run the bundled version of Spyder after installing it with Anaconda, the recommended method on Windows is to launch it via the Start menu shortcut.
+On other platforms, open Anaconda Navigator, scroll to Spyder under :guilabel:`Home` and click :guilabel:`Launch`.
 
 .. image:: /images/installation/installation-anaconda-navigator.png
    :alt: Anaconda Navigator showing Spyder
 
-If Spyder does not launch via this method or you prefer to use the command line, open Anaconda Prompt (Windows) or your terminal (other platforms), type ``conda activate base`` then ``spyder``.
+If Spyder does not start via this method or you prefer to use the command line, open Anaconda Prompt (Windows) or your terminal (other platforms), type ``conda activate base`` then ``spyder``.
+
 
+.. _install-conda-installing:
 
-New Conda environment
+Installing with Conda
 ~~~~~~~~~~~~~~~~~~~~~
 
-If you would like to have Spyder in a dedicated environment to update it separately from your other packages and avoid any conflicts, you can.
-Just run the following command in your Anaconda Prompt (Windows) or terminal (other platforms), for a minimal install of Spyder into a new environment called ``spyder-env``:
+.. note::
+
+   If using Mamba/Mambaforge, substitute ``mamba`` for ``conda`` in the following commands.
+
+With Miniconda/Miniconda/Miniforge, or to get a more reliable and up-to-date Spyder version with Anaconda, Spyder should be installed into its own dedicated Conda environment.
```

Comment:
```suggestion
With Miniconda/Mambaforge/Miniforge, or to get a more reliable and up-to-date Spyder version with Anaconda, Spyder should be installed into its own dedicated Conda environment.
```

Reply:
```suggestion
With Miniconda/Miniforge/Mambaforge, or to get a more reliable and up-to-date Spyder version with Anaconda, Spyder should be installed into its own dedicated Conda environment.
```

Good catch, but they should be in the correct order